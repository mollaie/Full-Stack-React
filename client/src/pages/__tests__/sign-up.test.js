import React from "react";
import renderer from "react-test-renderer";
import Signup from "../sign-up";

describe("sign-up test", () => {
  it("sign-up should match snapshot", () => {
    const component = renderer.create(<Signup setSignup={() => {}} />);
    console.log(component);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
