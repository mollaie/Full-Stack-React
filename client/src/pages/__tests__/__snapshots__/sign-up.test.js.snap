// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sign-up test sign-up should match snapshot 1`] = `
<div
  className="main-container"
>
  <img
    className=""
    height="250"
    src="login.svg"
    width="250"
  />
  <form
    onSubmit={[Function]}
  >
    <div
      className="item-container"
    >
      <label
        htmlFor="first_name"
      >
        First Name
      </label>
      <input
        name="first_name"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="Please enter your First Name"
        type="text"
        value=""
      />
    </div>
    <div
      className="item-container"
    >
      <label
        htmlFor="last_name"
      >
        Last Name
      </label>
      <input
        name="last_name"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="Please enter your Last Name"
        type="text"
        value=""
      />
    </div>
    <div
      className="item-container"
    >
      <label
        htmlFor="email"
      >
        Email
      </label>
      <input
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="Please enter your Last Name"
        type="email"
        value=""
      />
    </div>
    <div
      className="item-container"
    >
      <label
        htmlFor="username"
      >
        Username
      </label>
      <input
        name="username"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="Please enter your username"
        type="text"
        value=""
      />
    </div>
    <div
      className="item-container"
    >
      <label
        htmlFor="password"
      >
        Password
      </label>
      <input
        name="password"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="Please enter your password"
        type="password"
        value=""
      />
    </div>
    <div
      className="form-footer"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        type="submit"
      >
        Sing up
      </button>
      <button
        className="btn btn-secondary"
        onClick={[Function]}
        type="button"
      >
        Login
      </button>
    </div>
  </form>
</div>
`;
